bin/shorebird.bat

```diff
@@ -18,4 +18,4 @@ SET PowerShellScriptPath=%CurrentDirectory%shorebird.ps1
 
 REM Pass up to four arguments to the PowerShell script
 REM This number is arbitrary and can be increased if needed
-& %powershell_executable% -NoProfile -ExecutionPolicy Bypass -Command "& '%PowerShellScriptPath%' %1% %2% %3% %4%";
```

Comment:
Should we be concerned that `shorebird` on windows only passes along the first 4 arguments? :) @bryanoltman

Reply:
Yeah, there's probably a better way to do this :)

Reply:
I believe I was referencing https://superuser.com/a/1630441